setup.py

```diff
@@ -2,7 +2,36 @@
 
 import os
 import re
-from setuptools import setup, find_packages
+import sys
+import importlib
+from distutils.core import setup
+from setuptools import find_packages
+from setuptools.command.build_ext import build_ext
+from distutils.command.clean import clean
+
+# See https://gist.github.com/ctokheim/6c34dc1d672afca0676a for more details.
+try:
+    from Cython.Build import cythonize
+    language_level = "3"
+except ImportError:
+    cythonize = None
+
+# cython is not used unless specified.
+if cythonize:
```

Comment:
I've tried to run `python setup.py install --cython` but received an unrecognized option error, because cython was not installed. I would prefer, if a user sets the option but has no cython installed, that the user would receive a message, that she/he needs to install cython first.

Reply:
Yes. Change made and pushed.